README.md

```diff
@@ -13,6 +13,78 @@ IRC channel: https://freenode.net #googlebenchmark
 
 [Additional Tooling Documentation](docs/tools.md)
 
+
+## Installation Guide
+First make sure you have git and cmake installed (If not please install it)
+
+```
+sudo apt-get install git
+sudo apt-get install cmake
+```
+
+Now, let's clone the repository and build it
+
+```
+git clone https://github.com/google/benchmark.git
+cd benchmark
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=RELEASE
+make
+cmake ..  
+```
+
+We need to now place the built library to the location where our compiler can find it
+
+```
+cd ..
+sudo mkdir /usr/include/benchmark
+sudo cp include/benchmark/benchmark.h /usr/include/benchmark/
+sudo cp build/src/libbenchmark.a /usr/local/lib/
+```
+
+Let's test it now. We create a directory anywhere in our system
```

Comment:
i think you can remove the test instructions as that's covered by the rest of the docs already.

Reply:
It's 3 in the morning here. Will recommit the fix tomorrow

Reply:
Added the changes, as per told